cloudhub-2/modules/ROOT/pages/ch2-architecture.adoc

```diff
@@ -71,7 +71,7 @@ You can scale replicas vertically by selecting one of the available vCore sizes:
 [%header,cols="4*a"]
 |===
 |vCore Size | Total Memory | Heap Memory | Storage
-|0.1        |1.2 GB        |600 MB       | 8 GB
+|0.1        |1.2 GB        |500 MB       | 8 GB
```

Comment:
The actual value is the 40% of 1.2GB so it is closer to 480MB. Couldn't the 500MB be miss leading in case a customer actually want to use those extra 20MB.

Reply:
```suggestion
|0.1        |1.2 GB        |480 MB       | 8 GB
```